julia-mode/tryf

```diff
@@ -0,0 +1,13 @@
+# -*- mode: snippet -*-
+# name: try ... catch ... finally ... end
+# key: try
+# contributor: hankail05
+# --
+try
+    ${1:expr}
```

Comment:
Please add \`%\` before `${1:expr}` to give this snippet implicit selection support.

```diff
-  ${1:expr}
+  `%`${1:expr}
```